cmd/git_utils.go

```diff
@@ -159,7 +159,20 @@ func RunGitConfigLocalRemoteOriginURL(upstream string, dryrun bool) (string, err
 
 	upstreamStart := strings.Split(upstream, "/")[0]
 	kargs := []string{"config", "--local", "remote." + upstreamStart + ".url"}
-	return RunGit(kargs, dryrun)
+	remote, err := RunGit(kargs, dryrun)
+	if err != nil {
+		return remote, err
+	}
+
+	// Convert ssh remote to https
+	if strings.Contains(remote, "git@") {
+		remote = strings.Replace(remote, ":", "/", -1)
+		remote = strings.Replace(remote, "git@", "https://", -1)
+	}
+
+	remote = strings.Replace(remote, ".git", "", -1)
```

Comment:
The `-1` mean replace all occurrences. That makes me nervous. Probably unlikely, but what if the url contained `.git` or a `:` somewhere else? Can we limit these replacements to just the first or last occurrence?